docs/source/aws/examples/asg_not_used.rst

```diff
@@ -0,0 +1,84 @@
+AMI - ASG Garbage Collector
+====================================
+
+.. code-block:: yaml
+
+
+  - name: asg-mark-as-unused
+    resource: asg
+    mode:
+      type: periodic
+      schedule: "rate(24 hours)"
+      role: c7n_cloud_custodian_role
+      tags:
+        env: env
+        role: role
+        service: service
+        owner: owner
+        team: team
+    comments: |
+      Mark any unused ASG checking it every day.
+    filters:
+      - type: value
+        key: MinSize
+        value: 1
+        op: less-than
+      - type: value
+        key: DesiredCapacity
+        value: 1
+        op: less-than
+    actions:
+      - type: mark-for-op
+        op: delete
+        days: 30
+  - name: asg-unmark-as-unused
+    resource: asg
+    mode:
+      type: periodic
+      schedule: "rate(12 hours)"
+      role: c7n_cloud_custodian_role
+      tags:
+        env: env
+        role: role
+        service: service
+        owner: owner
+        team: team
+    comments: |
+      Unmark any ASG which has a value upper than 0.
+    filters:
+      - type: value
+        key: DesiredCapacity
+        op: greater-than
+        value: 0
+      - "tag:maid_status": not-null
+    actions:
+      - unmark
+  - name: asg-slack-alert
+    resource: asg
+    mode:
+      type: periodic
+      schedule: "rate(10 minutes)"
```

Comment:
this shouldn't really be a periodic policy if its a hot poller, i suggest removing the periodic from the whole set of policies.